olog.py

```diff
@@ -218,11 +241,12 @@ async def aput_property(self, property):
                                 json=property)
             property_from_server = res.json()
             property_from_server.pop('owner')
-            property.pop('owner')
-            property['attributes'] = sorted(property['attributes'],
-                                            key=lambda d: d['name'])
+            property_cp = property.copy()
+            property_cp.pop('owner')
+            property_cp['attributes'] = sorted(property_cp['attributes'],
+                                               key=lambda d: d['name'])
         if (not res.raise_for_status()) and \
-           (OrderedDict(property) != property_from_server):
+           (OrderedDict(property_cp) != property_from_server):
```

Comment:
Same comments as above regarding separating `raise_for_status()` into a separate statement, reworking the exception, and reconsidering the return value.

Also, does `httpx` give us an `OrderedDict` here? If so, does it do so in this case but not in logbook case?

Reply:
In **property** case, server puts to db and returns a ordered dictionary with a ordered list as value of  key `attributes` back. Other resource doesn't have this behavior.
I choose`OrderedDict` here just because it's easy to compare two dictionaries.